packages/amuse-aarsethzare/setup.py

```diff
@@ -26,16 +24,23 @@
 
 all_data_files = []
 
-packages = ['amuse.community.aarsethzare']
+packages = [
+    'amuse.community.aarsethzare',
+]
 
 package_data = {
 }
 
-mapping_from_command_name_to_command_class=setup_commands()
+mapping_from_command_name_to_command_class = setup_commands()
 
 setup(
     name=name,
-    version=version,
+    use_scm_version={
+        "root": "../..",
+        "relative_to": __file__,
+        "write_to": "src/amuse/community/aarsethzare/version.py",
```

Comment:
so this ends up in the package distribution?

Reply:
yes